### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by Zeukkari**
Remove react-router-dom dependency from header and helsinki logo -components.

### CODE REVIEW

File: src/common/externalLink/ExternalLink.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ export interface Props {
   to?: string;
   underline?: 'none' | 'hover' | 'always';
   variant?: 'default' | 'withArrow';
-  children: React.ReactNode;
+  children?: React.ReactNode;
```
Comment (Nizar-Rahme) [inline src/common/externalLink/ExternalLink.tsx]: `ReactNode` includes `null`s and `undefined`, so no need to make it optional.

File: src/common/header/Header.tsx [hunk 2]
```diff
@@ -1,12 +1,15 @@
 import React from 'react';
 
+import ExternalLink, { Props as LinkProps } from '../externalLink/ExternalLink';
 import HelsinkiLogo from '../helsinkiLogo/HelsinkiLogo';
 import styles from './header.module.scss';
 
-const Header: React.SFC = () => {
+const Header = ({ href }: LinkProps) => {
```
Comment (Nizar-Rahme) [inline src/common/header/Header.tsx]:
```suggestion
const Header: React.SFC<LinkProps> = ({ href }) => {
```

File: src/common/header/Header.tsx [hunk 3]
```diff
@@ -1,12 +1,15 @@
...
   return (
     <header className={styles.header}>
-      <HelsinkiLogo color="white" />
+      <ExternalLink href={href}>
```
Comment (Nizar-Rahme) [inline src/common/header/Header.tsx]: This shouldn't be an external link.
Unlike `react-router-dom` links, normal (external) links force the page to refresh.

Comment (Zeukkari) [reply src/common/header/Header.tsx]: do we really need routing here?

Comment (Nizar-Rahme) [reply src/common/header/Header.tsx]: I would say yes 🙂

**Pull request merged**